docs/tutorial/lambda-aws.txt

```diff
@@ -0,0 +1,121 @@
+============================================
+Deploy to Serverless AWS Lambda with BrefPHP
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. versionadded:: 1.17
+
+Overview
+--------
+
+`Bref <https://bref.sh>` allows to deploy serverless PHP applications on AWS Lambda.
+In this tutorial, you will deploy a simple PHP application with the MongoDB PHP extension,
+and connect to an Atlas cluster using AWS IAM authentication.
+
+Pre-requisites
+--------------
+
+Before you begin, you must install bref on your machine. You can follow the
+`official documentation to setup bref <https://bref.sh/docs/setup>`_.
+
+Install the mongodb extension
+-----------------------------
+
+By default, the bref layer is compiled with PHP and a few extensions. Additional extensions
+are provided in additional layers.
+
+Start by creating a new directory for your project and install the required mongodb
+and bref dependencies. This project will be a bare minimum PHP web application that
+connects to a MongoDB cluster, no framework is used, but you can adapt the code to
+your needs.
+
+.. code-block:: bash
+    mkdir bref-mongodb-app && cd bref-mongodb-app
+    composer init
+    composer require bref/bref bref/extra-php-extensions
+    vendor/bin/bref init
+
+
+A very simple web page have been created. To validate the deployment, you can start
```

Comment:
```suggestion
A very simple web page has been created. To validate the deployment, you can start
```

Note: docs guidelines may want to avoid using subjective terms like "very simple", but I'll defer to the docs team to revise the language.

Reply:
Regardless of docs team guidelines, I prefer avoiding such terms, especially in documentation aimed at beginners.

Reply:
```suggestion
The file ``index.php`` have been created. To validate the deployment, you can start
```

Reply:
```suggestion
The file ``index.php`` has been created. To validate the deployment, you can start
```